cmd/internal.go

```diff
@@ -147,15 +157,47 @@ func (r *ParseCommand) Run(assetPath string) error {
 
 	foundPipeline.Assets = nil
 
-	js, err := json.Marshal(struct {
+	result := struct {
 		Asset    *pipeline.Asset    `json:"asset"`
+		Lineage  *sqlparser.Lineage `json:"lineage"`
```

Comment:
this is not what we have in mind, instead we should be using the lineage value to fill out the columns of the asset, `Asset.Columns` field